art/tvu.bob

```diff
@@ -5,17 +5,18 @@
       |           |                                    |            |
       |           |                                    |            |  .------------.
       |           |                   .----------------+-------------->| Validators |
-      v           |  .-------.        |                |            |  `------------`
- .----+---.       |  |       |   .----+-------.   .----+---------.  |
- | Leader |--------->| Blob  |   | Retransmit |   | Replicate    |  |
- `--------`       |  | Fetch |-->|   Stage    |-->| Stage /      |  |
- .------------.   |  | Stage |   |            |   | Vote Stage   |  |
- | Validators |----->|       |   `------------`   `----+---------`  |
- `------------`   |  `-------`                         |            |
-                  |                                    |            |
-                  |                                    |            |
-                  |                                    |            |
-                  `------------------------------------|------------`
+      |           |                   |                |            |  `------------`
+      v           |  .-------.        |                |            `--------------------------.
+ .----+---.       |  |       |   .----+-------.   .----+---------.   .----+---.   .----+----.  |
```

Comment:
nit, there are extra plus signs on ledger write stage and storage stage where there are no vertical connections

Reply:
Good catch, removed those.